### PR TITLE
Demo condensed text

### PR DESCRIPTION
**Pull request by terrapanda**
image

### CODE REVIEW

File: lib/Bui.js [hunk 1]
```diff
@@ -140,6 +142,27 @@ class Bui extends PureComponent {
             <br />
             <xmp style={styles.xmp}>{strongCode}</xmp>
           </div>
+          <div style={styles.componentContainer}>
```
Comment (bucko13) [inline lib/Bui.js:146]: Could we add a header to match with the others? 
`<Header type="h3">Condensed Text Component</Header>`

**Review by bucko13 — CHANGES_REQUESTED** [review]

**Comment by terrapanda** [comment]
Should Condensed Text just fall under Text header since it is the same
component?

**Comment by bucko13** [comment]
Ah yeah, sorry I didn't see that in the demo and the rest was folded over. It looks a little weird to have the title for the section be the copy text when you've got other examples below. I think just having the text `Condensed Text Component` be as a normal text and leave the examples for the address snippets you've got underneath. The threshold option isn't even really doing anything in that example is it?

**Comment by terrapanda** [comment]
Threshold is ensuring the text doesn’t condense in that case. The idea is
that text could vary as a normal readable text span until it’s beyond a
threshold at which point it respects space.

Also, about to submit a PR so that if the text isn’t condensed then it
isn’t decorated with underline and cursor : ‘text’

The result is demonstrated below.

[image: image.png]

**Comment by terrapanda** [comment]
This way a user doesn't necessarily have to know the length of text
entering the component, only the max length they would like it to be.

**Comment by terrapanda** [comment]
image

**Comment by tynes** [comment]
That last image looks good to me. thoughts @bucko13 ?

**Comment by bucko13** [comment]
Very nice!

**Review by bucko13 — APPROVED** [review]

**Pull request merged**